### PR TITLE
Bug/tokens separators

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
- Update of the cell style
- Add new view to show separator
- Remove of the calculation for intents (UIKit constraints will do it for as)
- Set proper offset of the separator.

In this fix separators will be same in all iOS and devices versions.

### CODE REVIEW

File: Trust/Transactions/ViewControllers/TransactionsViewController.swift [hunk 1]
```diff
@@ -149,6 +150,11 @@ class TransactionsViewController: UIViewController {
         }, selector: #selector(Operation.main), userInfo: nil, repeats: true)
     }
 
+    private func showSeparator(for index: IndexPath) -> Bool {
```
Comment (vikmeup) [inline Trust/Transactions/ViewControllers/TransactionsViewController.swift]: @OlegGordiichuk do you think we should move this into `TableView`, let's have our own class for that.

File: Trust/Style/Styles.swift [hunk 2]
```diff
@@ -39,6 +39,7 @@ struct StyleLayout {
     struct TableView {
         static let heightForHeaderInSection: CGFloat = 30
         static let separatorColor = UIColor(hex: "d7d7d7")
+        static let separatorHeigh: CGFloat = 0.5
```
[the remark this answers is not in the document]
Comment (OlegGordiichuk) [reply Trust/Style/Styles.swift]: done

File: Trust/Transactions/ViewControllers/TransactionsViewController.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/Transactions/ViewControllers/TransactionsViewController.swift]: Done

**Comment by vikmeup** [comment]
@OlegGordiichuk is there any ways to make it work for iOS 11/12 by using native functionality? I'm not fan of custom solutions

**Pull request closed without merging**